omod/src/main/webapp/resources/scripts/htmlForm.js

```diff
@@ -40,6 +40,28 @@
         return jq(".submitButton.confirm").is(":disabled");
     }
 
+
+    /**
+     * Change the hours, minutes and seconds dropdowns to new date with client timezone
+     * also change the encounterDate datepicker current date to new date with client timezone
+     **/
+    var changeTimeWidgetClientTimeZone = function (clientDateTime) {
+        getField('encounterDate.value').datepicker('setDate', clientDateTime)
+        jq("#encounterDate").find(".hfe-hours").val(clientDateTime.getHours()).change();
+        jq("#encounterDate").find(".hfe-minutes").val(clientDateTime.getMinutes()).change();
+        jq("#encounterDate").find(".hfe-seconds").val(clientDateTime.getSeconds()).change();
+    }
+
+    /**
+     * Receives a datetime on ISO8601 format and return only the date part.
+     * Example: receives the datetime 2021-04-01T15:00:00.000+0000, the output is 04-01-2021.
+     **/
+    var splitDate = function (date) {
```

Comment:
I get scared whenever i see custom date manipulation functions. No JavaScript utility or library out there?

Reply:
I will use moment for that